src/main/java/frc/pathrecognizer/PixyCameraConnector.java

```diff
@@ -52,6 +54,29 @@ public PixyCameraConnector() {
             block.getAngle()
           );
 
+          switch  (count){
```

Comment:
too much duplication. there's no need for the count variable.
a simple call to block.getWidth(), block.getHeight() will be enough.
if you'd like to konw how many targets there are without counting them up, just log out the value of blocks.size()